eclair-core/src/main/resources/reference.conf

```diff
@@ -47,6 +47,7 @@ eclair {
     var_onion_optin = optional
     payment_secret = optional
     basic_mpp = optional
+	keysend = optional
```

Comment:
We shouldn't activate it by default until it is in the spec.
It should be opt-in since it may break once spec-ed (change in the feature bit used).

Reply:
Done